modules/ui/confirm.js

```diff
@@ -1,5 +1,7 @@
-iD.ui.confirm = function(selection) {
-    var modal = iD.ui.modal(selection);
+import { modal as modalModule } from './modal';
```

Comment:
I had to do a named import of the "modal" module in a couple of places because of collisions with the local `modal` variable (see below)… :unamused: